src/js/components/projects/detail.tsx

```diff
@@ -130,6 +184,35 @@ const ProjectDetail = (props: RouteComponentProps) => {
           { name: project.name },
         ]}
         onRenderHeaderActions={onRenderHeaderActions}
+        sidebar={[
+          <div
+            key="addmember-section"
+            className="slds-m-bottom_medium add-member"
+          >
+            <h2 className="slds-text-heading_medium slds-p-bottom_small">
```

Comment:
@mirisuzanne I want to note that the spacing between this and the title/description above it will be fixed once we merge the changes in from the task-pr-status branch since we are separating out the longform text into it's own div (I'll add margin under each section once it is merged).